pydantic/main.py

```diff
@@ -214,10 +215,11 @@ def __new__(mcs, name, bases, namespace):
                         config=config,
                     )
 
+            type_blacklist = TYPE_BLACKLIST + config.keep_untouched
```

Comment:
please rename `TYPE_BLACKLIST` to `UNTOUCHED_TYPES` to be consistent with config.

Reply:
Done :)